src/modules/app/constants/buildConfig.js

```diff
@@ -2,24 +2,54 @@
 
 import type { BuildConfigType } from '../../../../build-configs/configs/BuildConfigType'
 import loadBuildConfig from '../../../../build-configs'
-import { INTEGREAT_ICONS } from '../../../../build-configs/configs/integreat'
-import { MALTE_ICONS } from '../../../../build-configs/configs/malte'
-import integreatAppLogo from '../../../../build-configs/configs/integreat/assets/app-logo.png'
-import integreatLocationMarker from '../../../../build-configs/configs/integreat/assets/location-marker.svg'
-import malteAppLogo from '../../../../build-configs/configs/malte/assets/app-logo.svg'
-import malteLocationMarker from '../../../../build-configs/configs/malte/assets/location-marker.svg'
+import integreatAppLogo from '../../../../build-configs/assets/integreat/app-logo.png'
+import integreatLocationMarker from '../../../../build-configs/assets/integreat/location-marker.svg'
+import malteAppLogo from '../../../../build-configs/assets/malte/app-logo.svg'
+import malteLocationMarker from '../../../../build-configs/assets/malte/location-marker.svg'
+import integreatIntroLanguageIcon from '../../../../build-configs/assets/integreat/Language.svg'
+import integreatIntroEventsIcon from '../../../../build-configs/assets/integreat/Events.svg'
+import integreatIntroOffersIcon from '../../../../build-configs/assets/integreat/Offers.svg'
+import integreatIntroSearchIcon from '../../../../build-configs/assets/integreat/Search.svg'
+import malteIntroLanguageIcon from '../../../../build-configs/assets/malte/Language.svg'
+import malteIntroEventsIcon from '../../../../build-configs/assets/malte/Events.svg'
+import malteIntroOffersIcon from '../../../../build-configs/assets/malte/Offers.svg'
+import malteIntroSearchIcon from '../../../../build-configs/assets/malte/Search.svg'
+import { INTEGREAT_ASSETS, MALTE_ASSETS } from '../../../../build-configs/assets/AssetsType'
 
-export const buildConfigIconSet = (): {| appLogo: string, locationMarker: string |} => {
-  const iconSetString = buildConfig().iconSet
-  if (iconSetString === INTEGREAT_ICONS) {
+type IconSetType = {|
+  appLogo: string,
+  locationMarker: string,
+  intro: {
+    events: string,
+    language: string,
+    offers: string,
+    search: string
+  }
+|}
+
+export const buildConfigIconSet = (): IconSetType => {
+  const iconSetString = buildConfig().assets
```

Comment:
Either use the new term `assets` completely or stick to `iconSet`.

Reply:
done